src/lib/ams.c

```diff
@@ -32,6 +32,10 @@ DECLARE_TR_CTX(ams_tr, SOF_UUID(ams_uuid), LOG_LEVEL_INFO);
 
 static struct ams_context ctx[CONFIG_CORE_COUNT];
 
+/* AMS messages */
+const uint8_t LP_KEY_PHRASE_DETECTED_UUID[UUID_SIZE] =
+	"LP_KEYPHDET    ";
```

Comment:
is this supposed to be a UUID? This doesn't look like one? Although I don't know whether this would be a valid UUID, just usually they look differently

Reply:
I wanted to be aligned with AMS on ACE platform implementation. But I agree it doesn't look like valid UUID. I'll use `DECLARE_SOF_RT_UUID` instead